test/test_cwp.py

```diff
@@ -0,0 +1,52 @@
+# type: ignore
+from bpmncwpverify.cwp import Cwp
+from bpmncwpverify.state import SymbolTable
+from returns.pipeline import is_successful
+import os
+
+current_directory = os.path.dirname(os.path.abspath(__file__))
+
+cwp_path = os.path.join(current_directory, "example", "cwp.xml")
+
+
+def test_cwp_model_gen():
+    cwp = Cwp.from_xml(cwp_path)
+
+    assert is_successful(cwp)
+
+    cwp = cwp.unwrap()
+
+    mapping = {
+        "paymentOwner == buyerName && backpackOwner == sellerName": "Init_Purchase_Pending",
+        "terms == noRetry || paymentOffered == noRetryPayment": "Purchase_Failed",
+        "terms == agreed && paymentOffered == paymentAmount": "Purchase_Agreed",
+        "paymentOwner == sellerName && backpackOwner == buyerName": "Ownerships_Switched",
+        "terms != pending || paymentOffered != pendingPayment": "Negotiations",
+    }
+
+    for itm in cwp.edges.values():
+        assert itm.expression in mapping
+        assert itm.dest.name == mapping[itm.expression]
+
+
+def test_cwp_ltl_gen():
+    cwp = Cwp.from_xml(cwp_path)
+
+    assert is_successful(cwp)
+
+    cwp = cwp.unwrap()
+
+    state_path = os.path.join(current_directory, "example", "negotiation_state.txt")
```

Comment:
These should be in a `tests/resources/cwp` file rather than `example` directory. Let's separate out `test` input from `examples` as they serve two different purposes.